packages/cysync-core/src/hooks/useTabsAndDialog.ts

```diff
@@ -0,0 +1,92 @@
+import { ReactNode, useState } from 'react';
```

Comment:
This file already exists as `useTabsAndDialogs`

Reply:
done